de.tubs.variantsync.core/src/de/tubs/variantsync/core/patch/DeltaFactoryManager.java

```diff
@@ -63,5 +73,53 @@ public IDeltaFactory getFactoryByFile(IFile file) throws NoSuchExtensionExceptio
 	public static IDeltaFactory getDefaultFactory() {
 		return DefaultDeltaFactory.getInstance();
 	}
+	
+	@Override
+	public synchronized List<IDeltaFactory> getExtensions() {
+		if (extensionLoader != null) {
+			synchronized (extensions) {
+				if (extensionLoader != null) {
+					extensionLoader.loadProviders(this);
+					extensionLoader = null;
+				}
+			}
+		}
+		return Collections.unmodifiableList(extensions);
+		}
+	
+	protected void setExtensionLoaderInternal(IExtensionLoader<IDeltaFactory> extensionLoader) {
+		this.extensionLoader = extensionLoader;
+		}
+	
+	
+	
+	@Override
+	public synchronized boolean addExtension(IDeltaFactory extension) {
+	if (extension != null) {
+		for (final IDeltaFactory t : extensions) {
+			if (t.getId().equals(extension.getId())) {
+				return false;
+			}
+		}
+		if (extension.initExtension()) {
+			extensions.add(extension);
+			return true;
+		}
+	}
+	return false;
+}
```

Comment:
Same here, please fix the indentation by using the style config from FeatureIDE.